visual-migrator/src/app/verify-migration-ui.tsx

```
@@ -0,0 +1,136 @@
+                  </div>
+                  <div className="w-full md:w-1/3">
+                    <h3 className="text-lg font-semibold mb-2">Transformed</h3>
+                    <pre className="whitespace-pre-wrap break-words bg-gray-950 text-gray-100 p-3 rounded">
+                      {record.transformed.body}
+                    </pre>
+                  </div>
+                  <div className="w-full md:w-1/3">
+                    <h3 className="text-lg font-semibold mb-2">Transformed Body (Rendered HTML)</h3>
+                    <div
+                      className="bg-white text-gray-900 p-3 rounded shadow"
+                      dangerouslySetInnerHTML={{ __html: record.transformed.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)